.github/workflows/integration-test.yml

```diff
@@ -5,6 +5,8 @@ on: [push, pull_request]
 env:
   TEST_BROWSER_HEADLESS: 1
   CI: 1
+  OPENSEARCH_DASHBOARDS_VERSION: 2.0
+  OPENSEARCH_CORE_VERSION: 2.0
```

Comment:
This 2 variables need to be defined with `echo... >> $GITHUB_ENV` too to be available for reference.